net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/Downloads
+  - --filesystem=~/Music
+# Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+modules:
+  - shared-modules/linux-audio/jack2.json
+
+  - name: SonoBus
+    buildsystem: simple
+    build-commands:
+      - CONFIG=Release make -j$(nproc) -C Builds/LinuxMakefile Standalone
+      - install -Dm755 -t ${FLATPAK_DEST}/bin/ Builds/LinuxMakefile/build/sonobus
```

Comment:
As I mentionned, it's better if this and the lined after be moved to `post-install` instead of `build-commands`

Reply:
Yeah, I don't think separating `post-install` makes sense here.